aries_cloudagent/anoncreds/holder.py

```diff
@@ -259,6 +279,56 @@ async def store_credential(
 
         return credential_id
 
+    async def store_credential_w3c(
+        self,
+        credential_definition: dict,
+        credential_data: dict,
+        credential_request_metadata: dict,
+        credential_attr_mime_types: dict = None,
+        credential_id: str = None,
+        rev_reg_def: dict = None,
+    ) -> str:
+        """Store a credential in the wallet.
+
+        Args:
+            credential_definition: Credential definition for this credential
+            credential_data: Credential data generated by the issuer
+            credential_request_metadata: credential request metadata generated
+                by the issuer
+            credential_attr_mime_types: dict mapping attribute names to (optional)
+                MIME types to store as non-secret record, if specified
+            credential_id: optionally override the stored credential id
+            rev_reg_def: revocation registry definition in json
+
+        Returns:
+            the ID of the stored credential
+
+        """
+        try:
+            secret = await self.get_master_secret()
+            cred_w3c = W3cCredential.load(credential_data)
+            cred_w3c.to_legacy()
+            await asyncio.get_event_loop().run_in_executor(
+                None,
+                cred_w3c.process,
+                credential_request_metadata,
+                secret,
+                credential_definition,
+                rev_reg_def,
+            )
+            cred_recvd = Credential.from_w3c(cred_w3c)
```

Comment:
Eventually we want to store the credential in the W3C format in the wallet, so should add a "TODO" comment here.  (This will require changes to other endpoints that fetch credentials so prob shouldn't try to incorporate into this PR)